packages/bruno-app/src/utils/importers/postman-collection.js

```diff
@@ -54,7 +54,11 @@ const convertV21Auth = (array) => {
   }, {});
 };
 
-const importPostmanV2CollectionItem = (brunoParent, item, parentAuth, options) => {
+function sanitizeName(name) {
+  return name.replace(/\[|\]/g, ''); // This will remove square brackets
```

Comment:
This will not work for `Some / Environment / Name` as it'll result in `Some _ Environment / Name`; also, there are more forbidden characters. 

Furthermore, this is prone to introduce a silent crash if the user has two environments following the same pattern, e.g.  `Some / Env` and `Some [ Env` with results in the same name - `Some  Env`.

I proposed a detailed error instead to inform user of the invalid characters (https://github.com/usebruno/bruno/pull/2200)